Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -418,6 +420,23 @@ def get_logs_defender_download_request(self, hostname, lines):
         headers = self._headers
         return self._http_request('get', 'logs/defender/download', params=params, headers=headers, resp_type="content")
 
+    def get_file_integrity_events(self, hostname):
+        """
+        Get runtime file integrity audit events
+
+        Args:
+            hostname (str): The hostname for which to get runtime file integrity events
+
+        Returns:
+            HTTP response
+        """
+        endpoint = "audits/runtime/file-integrity"
+
+        headers = self._headers
+        params = assign_params(hostname=hostname, limit=10, sort="time", reverse=True)
```

Comment:
I see the limit here is a constant, can we make this an argument to make the command more adjustable to clients? if yes let's add this as an argument in the yml and set the default to 10.

Reply:
+1